### DISCUSSION TITLE
这是个bug么？

### DISCUSSION THREAD

**Discussion by maweijiao** [apache/doris#23787] [A - General / Q&A]
select *
from numbers("number"="10") a 
left join numbers("number"="10") b on 1=1 and b.number = 1
![image](https://github.com/apache/doris/assets/117417906/1fa9d7fa-6aea-4300-97c4-522daeb499e1)

**Comment by ixzc** [comment]
Which version?

**Reply by maweijiao** [reply]
1.2.3

**Reply by LemonLiTree** [reply]
we will fix it
